awx/main/credential_plugins/hashivault.py

```diff
@@ -89,7 +89,6 @@ def kv_backend(**kwargs):
             params['version'] = kwargs['secret_version']
         try:
             mount_point, *path = pathlib.Path(secret_path.lstrip(os.sep)).parts
-            '/'.join(*path)
```

Comment:
There's a decent enough chance we might be trying to catch a specific corner case here in this exception block (other than nested paths.)

Changing this to `'/'.join(path)' instead of deleting it should fix the bug without removing the check altogether.

Reply:
@jakemcdermott 

Thanks for the review.
Sure, that makes sense. I have done the requested change.

Reply:
@jakemcdermott

Thanks for the review.
Sure, that makes sense. I have done the requested change.